src/Dashboard/Grid.php

```diff
@@ -380,6 +382,23 @@ public function show(bool $mini = false, ?string $token = null)
             </span>
          </div>
 HTML;
+        } else if ($placeholder) {
+            $params = [
+                'placeholder_msg' => __('This is a placeholder dashboard. You can switch to a dashboard with your real data in the general setup or your user preferences.'),
+                'change_global_msg' => __('Change default dashboard'),
+                'change_my_msg' => __('Change my default dashboard'),
+                'change_global_url' => \Config::getFormURL() . '?forcetab=Config$2',
+                'change_my_url' => \Preference::getSearchURL() . '?forcetab=Config$1'
+            ];
+            $filters = TemplateRenderer::getInstance()->renderFromStringTemplate(<<<TWIG
+                <div class='placeholder_info' style="background-color: transparent; color: var(--tblr-body-color); font-size: var(--tblr-body-font-size)">
+                    <div class="alert alert-warning">{{ placeholder_msg }}</div>
+                    <button class="btn btn-primary btn-sm change-global-dashboard"
+                            onclick="window.location='{{ change_global_url }}'" role="link">{{ change_global_msg }}</button>
+                    <button class="btn btn-primary btn-sm change-my-dashboard"
+                            onclick="window.location='{{ change_my_url }}'" role="link">{{ change_my_msg }}</button>
+                </div>
+TWIG, $params);
```

Comment:
```suggestion
            $params = [
                'change_global_url' => \Config::getFormURL() . '?forcetab=Config$2',
                'change_my_url' => \Preference::getSearchURL() . '?forcetab=Config$1'
            ];
            $filters = TemplateRenderer::getInstance()->renderFromStringTemplate(<<<TWIG
                <div class='placeholder_info' style="background-color: transparent; color: var(--tblr-body-color); font-size: var(--tblr-body-font-size)">
                    <div class="alert alert-info m-2">
                        <div class="d-flex">
                            <i class="ti ti-info-circle fa-2x me-3"></i>
                            <div>
                                <h4 class="alert-title">
                                    {{ __('This is a placeholder dashboard.') }}
                                </h4>
                                <div class="text-secondary">
                                    {{ __(' You can switch to a dashboard with your real data in the general setup or your user preferences.') }}
                                </div>

                                <div class="mt-2">
                                    <a class="btn btn-info change-global-dashboard me-2"
                                    href="{{ change_global_url }}">
                                        <i class="ti ti-settings-down"></i>
                                        <span>{{ __('Change default dashboard') }}</span>
                                    </a>
                                    <a class="btn btn-info change-my-dashboard"
                                    href="{{ change_my_url }}">
                                        <i class="ti ti-user-cog"></i>
                                        <span>{{ __('Change my default dashboard') }}</span>
                                    </a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
TWIG, $params);
```

changes:
- use a more complex alert layout (title, sub text, "footer")
- use info actions class (instead the mix of warning/primary)
- add icons
- some spacing
- use href instead onclick+window.location (didn't understand this one)
- use inline gettext (not using passing params) in inline twig

![image](https://github.com/glpi-project/glpi/assets/418844/dc0aa37c-da90-4ed9-a764-6914f7f57edb)